backend/python/requirements.txt

```diff
@@ -50,6 +50,7 @@ pyasn1-modules==0.2.8
 pycparser==2.20
 pymongo==3.11.3
```

Comment:
do we ever use this?

Reply:
good catch - looks like its for mongodb, so no